src/common/translation/i18n/en.json

```diff
@@ -363,7 +363,10 @@
     "passwordCopySuccess": "Password was copied succesfully",
     "passwordLabel": "Your personal password:",
     "title": "Register for an event",
-    "unexpectedError": "An unexpected error took place"
+    "unexpectedError": "An unexpected error took place",
+    "noFreeTicketSystemPasswordsLeftLabel": "Unfortunately there are no more passwords left",
+    "noFreeTicketSystemPasswordsErrorTitle": "Oh no!",
+    "noFreeTicketSystemPasswordsErrorDescription": "Unfortunately this event's passwords have already run out.</br></br>Please go back and choose another event."
```

Comment:
Just a note, that the translations are still handled through a Google Spreadsheet. I will accept this, because I want to get rid of that spreadsheet :D More about this in a [Jira ticket ](https://helsinkisolutionoffice.atlassian.net/browse/KK-1043).

Reply:
You can blame me for that Spreadsheet. It was supposed to be a very temporary solution until we got a proper translation system up and running. 

For an open source project like this I would recommend taking a look at Weblate, it integrates well with GitHub and as I understand it is is free for open source project.

Reply:
Alright, we can discuss this e.g. tomorrow in refinement.